src/ttsnake.c

```diff
@@ -47,7 +47,7 @@
 #define SNAKE_TAIL	 '.'	 /* Character to draw the snake tail. */
 #define SNAKE_BODY       'x'     /* Character to draw the snake body. */
 #define SNAKE_HEAD	 '0'	 /* Character to draw the snake head. */
-#define ENERGY_BLOCK     '+'	 /* Character to draw the energy block. */
+#define ENERGY_BLOCK '+'	 /* Character to draw the energy block. */
```

Comment:
```suggestion
#define ENERGY_BLOCK     '+'	 /* Character to draw the energy block. */
```
Just maintaning consistency here.